components/brave_rewards/resources/internals/components/externalWallet.tsx

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Utils
+import { getLocale } from '../../../../common/locale'
+
+interface Props {
+  info: RewardsInternals.ExternalWallet
+}
+
+const getWalletStatus = (status: RewardsInternals.WalletStatus) => {
+  switch (status) {
+    case 0: {
```

Comment:
We can't use the definitions like `WalletStatus.NOT_CONNECTED` here?

Reply:
sadly no :(